main.py

```diff
@@ -29,7 +24,7 @@ def __init__(self, error):
 
 class AppMetaCache:
 
-    def __init__(self, host, port):
+    def __init__(self, host="127.0.0.1", port=6379):
```

Comment:
the `port` is already defaulted via the program argument `app-meta-cache-port`.

we can chat about this, but i a believe we should only hard-code a default 1 time, and we are already ensuring that the `args` are setup with `app-meta-cache-port` as `6379`.  so specifying it here is redundant.

But I think I understand why you did it, cause you wanted to default `host` to `127.0.0.1`.

it i just a bit odd that we default the program arguments to 
`wes-app-meta-cache` & `6379`
but default them in this function to 
`127.0.0.1` & `6379`

Reply:
That's a fair point. I can remove the default here, just so there's no implicit defaults. Basically, both main.py _and_ test.py will explicitly declare what they talk to by default.